Resources/Prototypes/_DV/CosmicCult/Alerts/entropy.yml

```diff
@@ -0,0 +1,18 @@
+- type: alert
+  id: CosmicEntropy
+  icons:
+  - sprite: /Textures/_DV/CosmicCult/Icons/alert_entropy.rsi
+    state: base0
+  alertViewEntity: AlertCosmicEntropySpriteView
+  name: alerts-cosmiccult-entropy-name
+  description: alerts-cosmiccult-entropy-desc
+
+- type: entity
+  id: AlertCosmicEntropySpriteView
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Sprite
+    sprite: /Textures/_DV/CosmicCult/Icons/alert_entropy.rsi
```

Comment:
/Textures/ prefix isnt needed for entity, maybe alert it shitcode and needs it though